### PR TITLE
Document unsupported reason 'virtualized image'

### PR DESCRIPTION
**Pull request by mdegat01**
## Proposed change

Document new unsupported reason added in https://github.com/home-assistant/supervisor/pull/4968

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/supervisor/pull/4968
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by mdegat01** [comment]
As the parent PR in supervisor not core, `current` is the correct branch

**Comment by mdegat01** [comment]
@agners I wrote a draft of this documentation. Figured you could either suggest changes or take it from here if we want more info on what specifically is wrong with the other images in this context or better solution instructions.

File: source/more-info/unsupported/virtualization_image.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Virtualized-Image"
```
Comment (agners) [inline source/more-info/unsupported/virtualization_image.markdown]:
```suggestion
title: "Virtualization in use"
```

File: source/more-info/unsupported/virtualization_image.markdown [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+description: "More information on why virtualized image marks the installation as unsupported."
+---
+
+## The issue
+
+Home Assistant OS publishes specific images which support being run on a virtualized
+system. These come with guest tools and other changes to make this easier. The 
+image of Home Assistant OS you are running on your virtualized system is not one
+of these.
```
Comment (agners) [inline source/more-info/unsupported/virtualization_image.markdown]:
```suggestion
Your Home Assistant OS installation appears to run in a virtualized environment using
a disk image which is not meant to run in a virtualized environment.
 
Home Assistant OS publishes specific images which support being run on a virtualized
system. These are optimized to run in virtual environments, e.g. have guest tools installed
and para-virtualization drivers enabled.
```

**Review by agners — CHANGES_REQUESTED** [review]

**Comment by caraar12345** [comment]
Hi folks! With [Supervisor 2024.04.0](https://github.com/home-assistant/supervisor/releases/tag/2024.04.0), this issue now fires, but because this documentation hasn't been deployed yet, the `Learn more` [link](https://www.home-assistant.io/more-info/unsupported/virtualization_image) 404s.

**Comment by agners** [comment]
> Hi folks! With [Supervisor 2024.04.0](https://github.com/home-assistant/supervisor/releases/tag/2024.04.0), this issue now fires, but because this documentation hasn't been deployed yet, the `Learn more` [link](https://www.home-assistant.io/more-info/unsupported/virtualization_image) 404s.

Right, it is on beta channel currently.

@mdegat01 can you check the review here so we can get it merged?

File: source/more-info/unsupported/virtualization_image.markdown (hunk 1 above)
Comment (mdegat01) [reply source/more-info/unsupported/virtualization_image.markdown]: Doesn't that title imply that virtualization isn't supported? I see that I can uses spaces here, I thought I couldn't for some reason. But I'm not sure I agree with that title. What about just "Virtualized image"?

Comment (agners) [reply source/more-info/unsupported/virtualization_image.markdown]: Hm, yeah good point. Maybe:

```suggestion
title: "Image not suited for virtualization"
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @mdegat01 :+1:

File: source/more-info/unsupported/virtualization_image.markdown (hunk 1 above)
Comment (mdegat01) [reply source/more-info/unsupported/virtualization_image.markdown]: After chatting we settled on "Incorrect image for virtualization"

**Review by agners — APPROVED** [review]

**Comment by agners** [comment]
LGTM, thanks!

File: source/more-info/unsupported/virtualization_image.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Incorrect image for virtualization"
+description: "More information on why virtualization image marks the installation as unsupported."
+---
+
+## The issue
+
```
Comment (c0ffeeca7) [inline source/more-info/unsupported/virtualization_image.markdown:7]: this section describes why something is not running. What is the symptom? What is not working, how do I know I'm having this particular issue.

Comment (agners) [reply source/more-info/unsupported/virtualization_image.markdown:7]: We show a link in the UI to this page. So the user gets redirected here when the issue is present on his system.

**Pull request merged**